### PR TITLE
add brands

### PR DESCRIPTION
**Pull request by gvonkoss**

### CODE REVIEW

File: models/version.js [hunk 1]
```diff
@@ -218,6 +219,24 @@ function initModel(app) {
...
+				const manifests = this.get('manifests') || {};
+				let brands = [];
+				const type = this.get('type');
+
+				if (manifests.origami && Array.isArray(manifests.origami.brands)) {
+					brands = manifests.origami.brands;
+
+				}
+
+				if (brands && type === 'module') {
```
Comment (notlee) [inline models/version.js]: If the component is not a module the `brands` key is set to `undefined` as `brands()` doesn't return?

Comment (notlee) [reply models/version.js]: Would it be better to not set the `brands` key for non-module components?

File: test/integration/seed/basic/component.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ exports.seed = async database => {
 		origami: {
 			name: 'o-mock-component',
 			origamiType: 'module',
+			brands: [
```
Comment (notlee) [inline test/integration/seed/basic/component.js]: A test for non-module components would be a great addition.

File: models/version.js (hunk 1 above)
Comment (rowanmanning) [reply models/version.js]: I'd rather always have keys, but possibly this should be set to `null` – this matches the behaviour for the `type` and `subType` properties

File: test/integration/routes/v1-repos-(id)-versions.test.js [hunk 3]
```diff
@@ -167,4 +167,7 @@ describe('GET /v1/repos/:repoId/versions', () => {
 
 	});
 
+	describe('repo is service', () => {
```
Comment (notlee) [inline test/integration/routes/v1-repos-(id)-versions.test.js]: Expect this blank test, looks good to me!

Comment (gvonkoss) [reply test/integration/routes/v1-repos-(id)-versions.test.js]: 🤦‍♀️

**Pull request merged**